src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,225 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    public static final String TEXT_SPACER = "    ";
+    public static final java.nio.file.Path path = java.nio.file.Paths.get(System.getProperty("user.dir"));
+    public static final ArrayList<Task> userLists = new ArrayList<>();
+
+    public static void main(String[] args) {
```

Comment:
This method is too long ([>30 LoC](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)). You can consider ways to abstract out functional blocks of code and make the [normal execution path more prominent](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent).